third_party/rust/repos.bzl

```diff
@@ -7,20 +7,24 @@ load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 def rust_repos():
     http_archive(
         name = "rules_rust",
-        sha256 = "531bdd470728b61ce41cf7604dc4f9a115983e455d46ac1d0c1632f613ab9fc3",
-        strip_prefix = "rules_rust-d8238877c0e552639d3e057aadd6bfcf37592408",
-        url = "https://github.com/bazelbuild/rules_rust/archive/d8238877c0e552639d3e057aadd6bfcf37592408.tar.gz",
+        sha256 = "5e2f59778ee496064b2d96182bc8aa916a0e34921124a359f740f51e5e5afc38",
+        strip_prefix = "rules_rust-be0d6ca492f64cc8d460f54f467925ef2753ed89",
+        url = "https://github.com/lowRISC/rules_rust/archive/be0d6ca492f64cc8d460f54f467925ef2753ed89.tar.gz",
```

Comment:
For these internal forks can you explain why they're worth maintaining in a comment (doing it here lets us keep the forks very short while still documenting the decision a Git graph.